paddle/CMakeLists.txt

```diff
@@ -48,93 +48,3 @@ set(POSTFIX ".so")
 if(WIN32)
   set(POSTFIX ".dll")
 endif()
```

Comment:
line 40-50 is no need here, and the comments should be moved to test/CMakeLists.txt